ztp/ran-crd/site-config-crd.yaml

```diff
@@ -78,21 +78,16 @@ spec:
                         description: BaseDomain is the base domain to which the cluster should belong.
                         type: string
                       clusterType:
-                        description: clusterType either standard or sno
+                        deprecated: true
+                        description: clusterType either standard or sno (Ignored: Computed based on number of master nodes defined)
```

Comment:
Open question to @serngawy and @imiller0 : Should I keep these fields in as "deprecated", or should I just remove them from the CRD altogether?

Reply:
Removed now.